content/en/docs/concepts/policy/pod-security-policy.md

```diff
@@ -548,6 +549,21 @@ for the default list of capabilities when using the Docker runtime.
 - *RunAsAny* - No default provided. Allows any `seLinuxOptions` to be
 specified.
 
+### AllowedProcMountTypes
+
+`allowedProcMountTypes` is a whitelist of allowed ProcMountTypes.
+Empty or nil indicates that only the `DefaultProcMountType` may be used. 
+
+`DefaultProcMount` uses the container runtime defaults for readonly and masked
+paths for /proc.  Most container runtimes mask certain paths in /proc to avoid
+accidental security exposure of special devices or information. This is denoted
+as the string `Default`.
+
+The only other viable ProcMountType is `UnmaskedProcMount`, which bypasses the 
```

Comment:
@jessfraz When you say "viable" do you mean that `"Unmasked"` is the only other choice than `"Default"`? If so, you can simply omit the work "viable"

Reply:
updated sorry for the delay!